Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -1119,11 +1119,15 @@ public Task ClearFilter( params string[] fieldNames )
     /// Keep in mind that this command will always trigger <see cref="FilteredDataChanged"/> even
     /// though not any data is actually changed.
     /// </remarks>
-    public void FilterData()
+    public async void FilterData()

```

Comment:
Is this `async void` going to be problematic in the future?

Reply:
I don't think so.
There's the known problem of lost stack trace and the code just running without actually waiting.

I would indeed prefer to change to **Task**, I'll schedule it for v2.0.